scripts/jenkins/ardana/ansible/deploy-caasp.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: Deploy CaaSP using heat templates
+  hosts: ardana-ccp-c0-m1-mgmt
+  gather_facts: False
+  include_vars: "provider.yml"
+  vars:
+    caasp_image_url: "http://download.suse.de/install/SUSE-CaaSP-3-GM/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+    caasp_image_path: "/tmp/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+  tasks:
+  - name: Install caasp-openstack-heat-templates
+    always_run: True
+    become: yes
+    zypper:
+      name: caasp-openstack-heat-templates
+      state: present
+
+  - name: Download CaaSP image
+    always_run: True
+    get_url:
+      url: "{{caasp_image_url}}"
+      dest: "{{caasp_image_path}}"
```

Comment:
same here on the retries.

nit: please also wrap your variables with spaces, like "{{ caasp_image_url }}".

Reply:
Done